activity_calendar/models.py

```diff
@@ -403,6 +403,11 @@ def end_date(self):
         normal_duration = self.parent_activity.end_date - self.parent_activity.start_date
         return self.start_date + normal_duration
 
+    @property
+    def participant_count(self):
```

Comment:
Property is not used in `get_json_from_activity_moment(..)` (activity_calendar -> api.py)